website/static/js/logTextParser.js

```diff
@@ -92,7 +93,7 @@ var LogPieces = {
     node: {
         view: function (ctrl, logObject) {
             var nodeObject = logObject.embeds.nodes;
-            if(paramIsReturned(nodeObject, logObject)){
+            if(paramIsReturned(nodeObject, logObject) && nodeObject.data[0]){
                 return m('a', {href: nodeObject.data[0].links.html}, nodeObject.data[0].attributes.title);
```

Comment:
I realize this is my code but it looks suspect when a specific index is used from a data. Can you check why the first item in the array is used?